### DISCUSSION TITLE
How do I comment under a group?

### DISCUSSION THREAD

**Discussion by amirave** [3urobeat/steam-comment-service-bot#114] [Q&A] [answered]
I couldn't find any way to comment on a group in the docs/wiki.
I did find a way to comment on a user using their ID, but only for profiles, not groups.

**Comment by 3urobeat** [comment, marked as answer]
Hey, the group comment functionality is currently only available in the latest beta version but will come to the master branch in version 2.11 which is still in development. 
If you want to try the latest beta version feel free to do that by heading over to the [beta-testing branch](https://github.com/HerrEurobeat/steam-comment-service-bot/tree/beta-testing).

**Reply by amirave** [reply]
Thank you! What's the command to do so? I don't think the docs are updated in the beta branch.

**Reply by 3urobeat** [reply]
Yeah the other branch doesn't have a separate documentation so the docs are always up to date with the master branch.  

You can either use the `!groupcomment` or `!gcomment` command.  
Syntax: `!gcomment number_of_comments groupid`  

For now you sadly need to use the groupid64 as I haven't implemented my [steamid-resolver](https://github.com/HerrEurobeat/node-steamid-resolver) library yet that can convert group links into the id (will be implemented in a future beta version and in the master version aswell ofc).

**Reply by 3urobeat** [reply]
Here is a quick tutorial on how to get the id if you don't know how:  
https://steamcommunity.com/sharedfiles/filedetails/?id=1344514370

**Reply by amirave** [reply]
Thank you!
